test/utils/fizzy_engine.cpp

```diff
@@ -46,13 +46,13 @@ bool FizzyEngine::parse(bytes_view input)
     return true;
 }
 
-bool FizzyEngine::instantiate()
+bool FizzyEngine::instantiate(bytes_view wasm_binary)
 {
     try
     {
-        m_instance = fizzy::instantiate(std::move(m_instance->module));
+        m_instance = fizzy::instantiate(fizzy::parse(wasm_binary));
     }
-    catch (const fizzy::instantiate_error&)
+    catch (...)
```

Comment:
Why not catch parser_error/instantiate_error? Do we want any other exception to be caught here? If so, would also do the same for `parse()`?

Reply:
This is resistant to changes in exceptions in fizzy. Still requires some debugging when this happens. This is also done for parse().

Reply:
I saw it was done for parse in a subsequent commit. It is fine as long as it is consistent.